core/src/test/java/org/sonarsource/sonarlint/core/mediumtest/StandaloneIssueMediumTest.java

```diff
@@ -380,8 +380,9 @@ public void simpleJavaWithExcludedRules() throws Exception {
       false);
 
     final Collection<RuleKey> excludedRules = Collections.singleton(new RuleKey("squid", "S106"));
+    final Collection<RuleKey> includedRules = Collections.singleton(new RuleKey("javascript", "S2424"));
```

Comment:
I'm surprised there is no assertion updated. Isn't it supposed to raise a few new issues?